tests/topotests/bgp_evpn_route_map_set/test_bgp_evpn_route_map_set.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
Please add SPDX-Identifier (license).. And a copyright.

Reply:
Done.